jupyterlab-base/scripts/install-system-files

```diff
@@ -15,3 +15,27 @@ install -m 0755 pyteco/teco.py /usr/local/bin/teco
 # The default terminal colors look bad in light mode.
 git clone https://github.com/seebi/dircolors-solarized.git
 cp dircolors-solarized/dircolors* /etc
+
+
+# Install typst.  It's not packaged for Debian, but there are binary releases
+# available on its GitHub page.
+
+# Get the tags; throw away the calver attempt and any rc tags;
+# extract the most recent version.
+typst_ver=$(
```

Comment:
While I don't have any strong feelings on this, any thoughts on if this is better than just pinning the version? That would give us a more deterministic and maybe less fragile approach than scrapping, but maybe you always want the latest version?